### ISSUE TITLE
Where can I find the documentation for the commands supported by the library?

### ISSUE THREAD

**Issue by ghilainm** [elupus/arcam_fmj#15]
I was searching for some CLI documentation but couldn't find any.

Could you also indicate me what is the right command to power an SA20 amp knowing the host and port of the amp?

**Comment by elupus** [comment]
Doubt there is such a command. Only way to power on that i know is via the serial port on the back, or IR blaster.

**Comment by pp81381** [comment]
I use the following commands for my PA720s...

```
echo "Rack Host On"
arcam-fmj client --host $RACK_HOST --port $PORT --command 0x00 --data 0x01

echo "Rack Host Off"
arcam-fmj client --host $RACK_HOST --port $PORT --command 0x00 --data 0x00
```

**Comment by ghilainm** [comment]
@pp81381 Thx :). Would be nice to have a small guide on how to use the module.

How do you use it by the way? For which use case?

**Comment by pp81381** [comment]
@ghilainm It's a bit convoluted.   I have a Raspberry Pi set up with an infrared hat.  I have configured lirc/irexec to run those commands on receipt of infrared commands from a 'fake' remote.   I've programmed the fake remote into my Harmony Hub which allows me to include powering the amps on and off as part of Scenes.

Most people probably use a daisy chained 12V trigger cable to turn the amps on and off but I found that caused nasty audible side effects so I opted for IP control.
